### ISSUE TITLE
Unable to spawn in local servers

### ISSUE THREAD

**Issue by squeek502** [fortressforever/fortressforever#367]
QuakingNutz on Discord gets a weird `Loading` popup after selecting a class, appears to die in the midmap, and then the map resets. Only happens in a local server. Video of the issue:

https://cdn.discordapp.com/attachments/402929975843487756/561423468848087043/2019-03-29_22-31-50_Trim.mp4

EDIT: More info:

- Happens after clicking to spawn, can sit in midmap dead indefinitely
- Console says `Dropped from server (Server shutting down)` and then `Spawn Server ff_2fort` after that `Loading` popup
- Doesn't appear to be coming from config files in this case (class configs are execed every spawn so a rogue `map ff_2fort` could have this sort of effect?)

**Comment by ghost** [comment]
haha

**Comment by doublah** [comment]
Likely caused by maxplayers cvar (set to 1 to reproduce).

Is maxplayers 1 by default?

**Comment by hanoicity900** [comment]
moi nguoi vao day ung ho minh nhe
http://www.dackhuvietnam.com/2019/01/ban-biet-thu-lien-ke-van-canh-an-lac.html

**Comment by pvorisek25** [comment]
Yes, it happens when the sum of connected players is greater than the 'maxplayers' ConVar. Obviously, if you create a local server with the "Max. players" setting set to 0, the game will behave in the exact same way as seen in the video.
